website/static/js/osfHelpers.es6.js

```diff
@@ -372,6 +372,50 @@ var decodeText = function(text) {
     return text;
 };
 
+var parseName = function(text) {
+    const suffixList = ['jr','jnr','sr','snr','dr','phd','i','ii','iii','iv','v','md','esq'];
```

Comment:
While not actually an issues, suffixList is kind of a humorous name since this is JS and it is an array. 😆 

Please add a brief comment that explains the rationale for this list (e.g., it is difficult to know if something without a period in it is a suffix, so catch most by using data)